### DISCUSSION TITLE
Searing for viral but bacteria show up

### DISCUSSION THREAD

**Discussion by javad30** [ablab/spades#1163] [Q&A]
### Description of bug

Hi all, 

I did cDNA synthesis to amplify both RNA and DNA viruses and I used NexteraXT (Illumina) to amplify the vrial commmunities and and sequencing with NextSeq500 using a 75cycle kit with paired end 40nt reads and 6nt+6nt dual indexing.
I check the sequence data using Kraken and I can see the viral communities as we spiked some of the samples with known viruses and the data makes sense. However, I was also interested into metaviralspades.py. I was able to generate the data but when I blast the contigs/scafold only pseudomonas aeruginosa shows up (which is a bacteria). I was wondering if I can use spades for my sequence data (as it is 40bp). 

I attached the log and parameter file. 

I upload my scaffold file too. 

### spades.log

[spades.log](https://github.com/ablab/spades/files/12235794/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/12235796/params.txt)
[scaffolds.txt](https://github.com/ablab/spades/files/12235798/scaffolds.txt)

### SPAdes version

3.15.4

### Operating System

Ubuntu 22.04.2 LTS 

### Python Version

3.10.11

### Method of SPAdes installation

Manual 

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment]
This is perfectly expected. metaviralSPAdes outputs lots of non-viral contigs by design (see paper for more information). Also we strongly recommend to check final contigs with either viralVerify tool or with CheckV.

Also, note that metaviralSPAdes produces only complete circular (or linear with inverted ending repeats) contigs. If something was not assembled, it will not present in the output
